app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,22 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors
+
+    @ordered_actors = @movie.order_actors
+    @average_age_of_cast = @movie.average_age
+  end
+
+  def update
+    @movie = Movie.find(params[:id])
+
+    if params[:search].present?
+      @actor = Actor.find(params[:search])
+      @movie.add_actor(@actor)
```

Comment:
Inside this `update` method we're adding an actor. When I look at the method you wrote called `.add_actor`, it's doing `actors << @actor`, which is the same as calling `MovieActor.create!(movie_id: ..., actor_id: ...)`. 
So, we're not really updating a movie though, are we? This means it would fit better inside a controller action called `create`; and we're not creating a movie, we're creating a `MovieActor`, so maybe it should live in a `MovieActorsController` instead. 

[This article](http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/) has been helpful when figuring out the SRP of controllers and different approaches we can take in Rails.